### PR TITLE
feat: alby banner

### LINKED ISSUE #138: Support LNURL successActions
https://github.com/lnurl/luds/blob/luds/09.md

### PR DESCRIPTION
**Pull request by reneaaron**
- adds (internal) `amount` param to lnurl screen
 - fixes #138

### CODE REVIEW

**Comment by rolznz** [comment]
@reneaaron where can I test the success action?

File: components/AlbyBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { View } from "react-native";
+import { Text } from "~/components/ui/text";
+import { Button } from "./ui/button";
+import { router } from "expo-router";
+
+function AlbyBanner() {
+    const showAlbyBanner = new Date().getDate() === 21;
```
Comment (rolznz) [inline components/AlbyBanner.tsx:7]: This shows even after you pay, I feel like this is a bit strange

File: pages/send/PaymentSuccess.tsx [hunk 2]
```diff
@@ -30,6 +36,31 @@ export function PaymentSuccess() {
...
+          <View className="flex flex-col gap-2 items-center">
+            <Text className="text-muted-foreground text-center font-semibold2">
+              Message From Receiver
+            </Text>
+            {lnurlSuccessAction.tag == "message" &&
+              <Text className="text-foreground text-center text-2xl font-medium2">
+                {lnurlSuccessAction.message}
+              </Text>
+            }
+            {lnurlSuccessAction.tag == "url" &&
```
Comment (rolznz) [inline pages/send/PaymentSuccess.tsx:49]: is this correct? you have a same inner check too

Comment (reneaaron) [reply pages/send/PaymentSuccess.tsx:49]: The `tag` defines which type the successAction has. If the tag is "url", we use the `url` field from the SuccessAction. :thinking:

**Pull request merged**